charts/templates/rbac.yaml

```diff
@@ -0,0 +1,110 @@
+{{ if .Values.rbac.enabled }}
+apiVersion: rbac.authorization.k8s.io/{{ required "A valid .Values.rbac.apiVersion entry required!" .Values.rbac.apiVersion }}
+kind: Role
+metadata:
+  name: {{ template "druid-operator.fullname" . }}
+  labels:
+    app: "{{ template "druid-operator.fullname" . }}"
+    chart: "{{ include "druid-operator.chart" . }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+rules:
```

Comment:
I think this needs a rule for `ingresses.extensions` reflected by [this change](https://github.com/druid-io/druid-operator/commit/0372b5d17b82d2a823317f5b280c32681738ed17). Otherwise, attempts by the operator to create  workload objects will fail.